site/src/pages/DeploySettingsPage/optionValue.ts

```diff
@@ -41,7 +41,7 @@ export function optionValue(
       const experimentMap: Record<string, boolean> | undefined =
         additionalValues?.reduce(
           (acc, v) => {
-            return { ...acc, [v]: option.value.includes("*") ? true : false };
+            return { ...acc, [v]: option.value.includes("*") };
           },
           {} as Record<string, boolean>,
         );
```

Comment:
Didn't realize we had code like this. The `includes` change is good, but I think there's room to go two steps further, depending on what we want to do:

The types can be cleaned up:
```ts
const experimentMap = inputs?.reduce<Record<string, boolean>>((acc, v) => {
  return { ...acc, [v]: option.value.includes('*') };
}, {});
```
And then the spread operator can be replaced with a "mutation":
```ts
And depending on how many elements we expect to deal with, we can replace the `.reduce` call with a loop. The main issue with the spread syntax is that while it's clean looking,